11ty/_includes/dex-page.hbs

```diff
@@ -23,12 +23,6 @@ layout: base
     margin-left: 2ch;
   }
 
-  @media (max-width: 92em) {
```

Comment:
I think I get the intention (to keep things flush left and not waste screen space), but trust me when I say having the margin up against the edge of your phone just feels bad